workflows/differentialabundance.nf

```diff
@@ -217,10 +215,12 @@ workflow DIFFERENTIALABUNDANCE {
         }
         .unique()
 
-    ch_all_matrices = VALIDATOR.out.fom
-        .combine(ch_processed_matrices)                         // Add processed marices to what we have in the FOM
+    ch_all_matrices = VALIDATOR.out.sample_meta              // meta, samples
```

Comment:
Could you align these comments? Apologies, the GitHub suggestions editor misaligned them.